README.md

```diff
@@ -541,7 +541,7 @@ The path to the file containing the template to install.
 
 ### templates
 
-Specify index templates in form of hash. Can contain multiple templates.
+Specify index templates (to be created on startup) in the form of a Ruby hash (accepts JSON dict). Can contain multiple templates.
```

Comment:
Hash should be obvious for HashMap for normal Rubyists. We should revert this part.

Reply:
I came to this documentation as a user of fluentd (and this plugin) without having any Ruby experience - personally I wouldn't expect someone using (not developing) something written in language X to necessarily know about language X, what do you think?

Even once I eventually realised that 'hash' needed interpreting in the context of ruby it still wasn't obvious to me that a JSON dict string was the appropriate form to provide.

Reply:
Even of newbie for Ruby and Fluentd, I didn't accept your proposed change. Because, hash should be used in everywhere for Ruby world. So, I would like to insist revert that part.

Reply:
Ok I will remove the word "Ruby" and leave it just as "hash" - I think the other changes on this line (clarifying when it is created, and what format it is in) are valuable so before I push more commits, are you happy with this?
```
Specify index templates (to be created on startup) in the form of a hash (accepts JSON dict). Can contain multiple templates.
```

Reply:
Yes. I love to use your suggestion. 😉